tests/integration/nm/ovs_test.py

```diff
@@ -230,7 +230,9 @@ def _delete_iface(devname):
         if nmdev:
             nm.device.delete(nmdev)
         else:
-            nm.device.delete_connection(devname)
+            con_profile = nm.connection.ConnectionProfile()
```

Comment:
The previous commit moved the implementation but kept the API in order to avoid moving/changing too many parts at once. This commit removed the previous API and uses the new one.

When refactoting in small steps, this approach helps to keep changes to the minimum and makes sure nothing breaks at each step. It would have been clearer if there where more usages to the profile deletion action.